lewis/scripts/control.py

```diff
@@ -48,7 +48,7 @@ def show_api(remote, object_name):
             raw_value = str(getattr(obj, prop))
             value_lines = raw_value.split('\n')
 
-            current_value = value_lines[0][40:] + (
+            current_value = value_lines[0][:40] + (
```

Comment:
What you describe is the behaviour I was getting before making this change... it's exactly what it's supposed to fix (`:40` means "from beginning to index 40", `40:` means "from index 40 to end"). 

Are you sure you're using the correct version of lewis-control? 

If you have lewis installed, you have to be careful to use `$ ./lewis-control.py`, and not simply `$ lewis-control`... otherwise you're using the installed version without this change. I think this may also be how this slipped passed us in the first place?